### PR TITLE
Make bpkg install properly on MSYS

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ $ clib install bpkg/bpkg
...
 ### 3. Source Code
 
-To directly install `bpkg` from it's source code you have to clone it's repository and install with `make`:
+To directly install `bpkg` from it's source code you have to clone it's repository and run the `install` script:
 
 ```sh
 $ git clone https://github.com/bpkg/bpkg.git
 $ cd bpkg
-$ make install
+$ ./install
```
Comment (ccarpita) [inline README.md]: Minor nit: would prefer an extension on `install`

File: README.md [hunk 2]
```diff
@@ -87,8 +87,8 @@ $ bpkg install jwerle/suggest.sh@0.0.1 -g
 
 **Note:** to do that the packages **must be tagged releases** on the repository.
 
-You can also *installing packages without a `package.json`*.
-As long as there is a `Makefile` in the repository it will try to invoke `make install` so long as the `-g` or `--global` flags are set when invoking `bpkg install`.
+You can also *install packages without a `package.json`*.
+As long as there is a `Makefile` in the repository it will try to invoke `make install` as long as the `-g` or `--global` flags are set when invoking `bpkg install`.
```
Comment (ccarpita) [inline README.md]: +1

File: install.sh [hunk 3]
```diff
@@ -52,13 +52,55 @@ setup () {
...
+[ -z "$PREFIX" ] && PREFIX="/usr/local"
+
+# All 'bpkg' supported commands
+CMDS="json install package term suggest init utils update list show"
+
+make_install () {
+  make_uninstall
+  echo "  info: Installing $PREFIX/bin/$BIN..."
+  install -d $PREFIX/bin
+  source=$(<$BIN)
```
Comment (ccarpita) [inline install.sh]: `local source`

File: install.sh [hunk 4]
```diff
@@ -52,13 +52,55 @@ setup () {
...
+    [ -f "$source" ] && install $source $PREFIX/bin/$BIN-$cmd || install $BIN-$cmd $PREFIX/bin
+  done
+  return $?
+}
+
+make_uninstall () {
+  echo "  info: Uninstalling $PREFIX/bin/$BIN..."
+  rm -f $PREFIX/bin/$BIN
+  for cmd in $CMDS; do
+    rm -f $PREFIX/bin/$BIN-$cmd
```
Comment (ccarpita) [inline install.sh]: Recommend adding quotes around expanded terms for safety

**Comment by ccarpita** [comment]
Looks good overall, just some minor sanity stuff

**Comment by sschuberth** [comment]
Thanks for the suggestions, I've addressed them and force-pushed a new iteration of the PR.

**Pull request merged**